monitoring.md

```diff
@@ -0,0 +1,206 @@
+# Monitoring `amppackager` in production
+
+Once you've run `amppackager` server in production, you may want to monitor its
+health and performance, as well as the performance of the underlying requests to
+the AMP document server. 
+
+The sections below talk about the things you can monitor, and how to do it.
+
+All the command line examples are for Linux OS.
+
+## Monitoring health
+
+To check `amppackager`'s health, `curl` its `/healthz` endpoint:
+
+```
+$ curl https://localhost:8080/healthz
+```
+
+If the server is up and has a fresh, valid certificate, it will respond with
+`ok`. If not, it will provide an error message.
+
+## Monitoring performance
+
+You can take a step further and check a few performance metrics, both for
+requests handled by `amppackager`, and for the underlying gateway requests that
+`amppackager` sends to the AMP document server. Get all the metrics by `curl`ing
+the `/metrics` endpoint. 
```

Comment:
You should mention somewhere in this doc that this is a Prometheus endpoint, and that the preferred way of querying these metrics is to set up all amppkg replicas as targets and [query](https://prometheus.io/docs/prometheus/latest/querying/basics/) the prom server or [draw graphs](https://prometheus.io/docs/visualization/grafana/). (Or whatever the correct lingo is.)

Reply:
Done.